src/main/java/teammates/ui/controller/InstructorHomePageData.java

```diff
@@ -26,5 +34,137 @@ public String getInstructorFeedbacksPageLinkForCourse(String courseID) {
+                       delete, disabled, disabled);
+        addAttributeIf(true, delete, "onclick", "return toggleDeleteCourseConfirmation('" + courseId + "')");
+        
+        int numberOfPendingCommentsForThisCourse = numberOfPendingComments.get(courseId);
+        if (numberOfPendingCommentsForThisCourse <= 0) {
+            return Arrays.asList(enroll, view, edit,add, archive, delete);
+        } else {
+            String pendingGraphic = "<span class=\"badge\">" + numberOfPendingCommentsForThisCourse + "</span>"
+                                    + "<span class=\"glyphicon glyphicon-comment\"></span>"
+                                    + "<span class=\"glyphicon glyphicon-arrow-right\"></span>"
+                                    + "<span class=\"glyphicon glyphicon-envelope\"></span>";
+            ElementTag pending = createButton(
+                    pendingGraphic,
+                    getInstructorClearPendingCommentsLink(courseId),
+                    "Send email notification to recipients of " + numberOfPendingCommentsForThisCourse
+                        + " pending " + (numberOfPendingCommentsForThisCourse > 1 ? "comments" : "comment"));
+    
+            return Arrays.asList(enroll, view, edit,add, archive, pending, delete);
+        }
+    }
+    
+    private List<CourseTableSessionRow> createSessionRows(List<FeedbackSessionAttributes> sessions,
+            InstructorAttributes instructor, String courseId) {
+        List<CourseTableSessionRow> rows = new ArrayList<CourseTableSessionRow>();
+        int displayedStatsCount = 0;
+        
+        for (FeedbackSessionAttributes session : sessions) {
+            String name = PageData.sanitizeForHtml(session.feedbackSessionName);
+            String tooltip = PageData.getInstructorHoverMessageForFeedbackSession(session);
+            String status = PageData.getInstructorStatusForFeedbackSession(session);
+            String href = getFeedbackSessionStatsLink(session.courseId, session.feedbackSessionName);
+            
+            String recent = "";
+            if (session.isOpened() || session.isWaitingToOpen()) {
+                recent = " recent";
+            } else if (displayedStatsCount < InstructorHomePageData.MAX_CLOSED_SESSION_STATS
+                       && !TimeHelper.isOlderThanAYear(session.createdTime)) {
+                recent = " recent";
+                ++displayedStatsCount;
+            }
+            
+            String actions = "";
+            try {
+                actions = getInstructorFeedbackSessionActions(session, false, instructor,
+                        getCourseIdSectionNamesMap(sessions).get(courseId));
+            } catch (EntityDoesNotExistException e) {
+                // nothing
+            }
+            
+            rows.add(new CourseTableSessionRow(name, tooltip, status, href, recent, actions));
+        }
+        
+        return rows;
+    }
 }
```

Comment:
I didn't change this particular segment, but this is now done with commit 1adb81c.
A review would be appreciated.